### ISSUE TITLE
Firefox styling error in v465

### ISSUE THREAD

**Issue by ayumi-cloud** [octobercms/october#4964]
Running some tests in several browsers with the latest update and come across this error in v465 in Firefox.

See screenshot:

![image](https://user-images.githubusercontent.com/57409060/75785548-d4b10700-5d5b-11ea-8b55-0d49526e0869.png)

When we adjust css and remove the following code lines:

```css
a:focus {
    outline: thin dotted;
    outline: 5px auto -webkit-focus-ring-color;
    outline-offset: -2px;
}
```

Location: https://github.com/octobercms/october/blob/aff120fcec390309e93eae8f195f253623a5ea12/modules/system/assets/ui/storm.css#L93

and

https://github.com/octobercms/october/blob/aff120fcec390309e93eae8f195f253623a5ea12/modules/system/assets/ui/less/site.reset.less#L64-L66

We get the following result:

![image](https://user-images.githubusercontent.com/57409060/75785881-56089980-5d5c-11ea-8ee7-8762080a4759.png)

> The 2nd focus ring (white dotted line) has been removed.

Long story short - permission to remove this offending css code?
